extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.tsx

```diff
@@ -326,7 +184,7 @@ TrackedCornerstoneViewport.defaultProps = {
 
 const _viewportLabels = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I'];
```

Comment:
Same issue as the SR viewport, but I guess this one is less generic, so it doesn't matter, but the SR viewport is designed to be generically used in any HP.